Tests/ResultTests.swift

```diff
@@ -277,4 +280,194 @@ class ResultTestCase: BaseTestCase {
             XCTFail("flatMap should preserve the failure error")
         }
     }
+    
+    // MARK: - map/flatMapError Tests
```

Comment:
Addressed in [9bf29](https://github.com/Alamofire/Alamofire/pull/2135/commits/131354b48086ea94f3b3d9303347d2f64079bf29).